docs/reference/contributing/guidelines/workflow.md

```diff
@@ -8,7 +8,7 @@ All code changes and additions to Mbed OS are handled through GitHub. If you wan
 
 Before contributing an enhancement (new feature, new port and so on) please [discuss it on the forums](https://os.mbed.com/forum/) to avoid duplication of work, as we or others might be working on a related feature.
```

Comment:
Lets create a specific forum for this ahead of time and link directly

Reply:
So we already have 
https://os.mbed.com/forum/mbed/
https://os.mbed.com/forum/bugs-suggestions/

Are we saying we should also have something like
https://os.mbed.com/forum/new-features  ?

Reply:
Or should we just link directly to one of the latter two forums ?

Reply:
I think bugs and suggestions is good.